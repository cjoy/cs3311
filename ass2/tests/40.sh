# Student 3233283 studied 3648,SENGA1 starting in 08s2
php progress 3233283 13s2
