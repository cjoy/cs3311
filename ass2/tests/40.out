Kirsty Everest (3233283)

Course   Term Title                           Mark Grade   UOC
-------- ---- -----                          ----- ----- -----
COMP1917 08s2 Computing 1                       61    PS     6
     ... Level 1 Core Courses
INFS1603 08s2 Business Databases                65    CR     6
     ... Free elect
MATH1131 08s2 Mathematics 1A                    80    DN     6
     ... Level 1 Core Courses
SENG1031 08s2 Software Eng Workshop 1           71    CR     6
     ... Level 1 Core Courses
MATH1231 09x1 Mathematics 1B                    71    CR     6
     ... Level 1 Core Courses
COMP2111 09s1 System Modelling and Design       57    PS     6
     ... Level 2 Core Courses
INFS2603 09s1 Business Systems Analysis         77    DN     6
     ... Free elect
MATH1081 09s1 Discrete Mathematics              80    DN     6
     ... Level 1 Core Courses
SENG2010 09s1 Software Eng Workshop 2A          83    DN     3
     ... Fits no requirement. Does not count
ACCT1501 09s2 Accounting & Financial Mgt 1A     71    CR     6
     ... Fits no requirement. Does not count
COMP1927 09s2 Computing 2                       53    PS     6
     ... Level 1 Core Courses
COMP3711 09s2 Software Project Management       78    DN     6
     ... Level 3/4 Electives
MATH2859 09s2 Prob, Stats and Information       62    PS     3
     ... Level 2 Core Courses
SENG2020 09s2 Software Eng Workshop 2B          77    DN     3
     ... Fits no requirement. Does not count
ACCT1511 10x1 Accounting & Financial Mgt 1B     70    CR     6
     ... Fits no requirement. Does not count
ECON1101 10x1 Microeconomics 1                  75    DN     6
     ... Fits no requirement. Does not count
COMP2121 10s1 Microprocessors & Interfacing     71    CR     6
     ... Level 2 Core Courses
COMP2911 10s1 Eng. Design in Computing          83    DN     6
     ... Level 2 Core Courses
COMP3331 10s1 Computer Networks&Applications    75    DN     6
     ... Level 3 Core Courses
SENG3010 10s1 Software Eng Workshop 3A          93    HD     3
     ... Fits no requirement. Does not count
COMP2041 10s2 Software Construction             68    CR     6
     ... Level 2 Core Courses
COMP3421 10s2 Computer Graphics                 53    PS     6
     ... Level 3/4 Electives
COMP3511 10s2 Human Computer Interaction        58    PS     6
     ... Level 3/4 Electives
SENG3020 10s2 Software Eng Workshop 3B          72    CR     3
     ... Fits no requirement. Does not count
COMP3141 11s1 Software Sys Des&Implementat'n    60    PS     6
     ... Level 3 Core Courses
COMP3311 11s1 Database Systems                  68    CR     6
     ... Level 3 Core Courses
COMP9321 11s1 Web Applications Engineering      75    DN     6
     ... Fits no requirement. Does not count
SENG4921 11s1 Professional Issues and Ethics    73    CR     6
     ... Level 4 Core Courses
COMP3171 11s2 Object-Oriented Programming       47    PC     6
     ... Level 3/4 Electives
COMP9322 11s2 Service-Oriented Architectures    81    DN     6
     ... Fits no requirement. Does not count
SENG4910 11s2 Thesis Part A                           SY     6
     ... Level 4 Core Courses
COMP3121 12s1 Algorithms & Programming Tech     71    CR     6
     ... Level 3/4 Electives
COMP3411 12s1 Artificial Intelligence           50    PS     6
     ... Level 3/4 Electives
SENG4911 12s1 Thesis Part B                     75    DN    12
     ... Level 4 Core Courses
              Overall WAM                       69         195
TO DO
-----
36 UOC so far; need 6 UOC more
     ... Level 1 Core Courses
27 UOC so far; need 15 UOC more
     ... Level 2 Core Courses
18 UOC so far; need 6 UOC more
     ... Level 3 Core Courses
0 UOC so far; need 12 UOC more
     ... GE
